cmd/README.md

```diff
@@ -0,0 +1,521 @@
+# HarbourBridge: Spanner Evaluation and Migration
+
+[![cloudspannerecosystem](https://circleci.com/gh/cloudspannerecosystem/harbourbridge.svg?style=svg)](https://circleci.com/gh/cloudspannerecosystem/harbourbridge)
+
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation and
+migration, using data from an existing PostgreSQL, MySQL or DynamoDB database. 
+The tool ingests schema and data from either a pg_dump/mysqldump file or directly 
+from the source database, and supports both schema and data migration. For schema
+migration, HarbourBridge automatically builds a Spanner schema from the schema
+of the source database. This schema can be customized using the HarbourBridge
+schema assistant. For data migration, HarbourBridge creates a new Spanner
+database using the Spanner schema built during schema migration, and populates
+it with data from the source database.
+
+For more details on schema customization and use of the schema assistant, see
+[web/README](web/README.md). The rest of this README describes the command-line
```

Comment:
The web/README link seems to be broken.

Reply:
These copies in cmd/ are only supposed to be used as a staging area until we are ready to copy them over to their original locations. Thus, it's okay if the links are temporary broken. We will get an opportunity to review the same again in a separate PR focused on updating READMEs. I will add a temporary banner to top of these READMEs to reflect the same.